goosebit/db/migrations/models/0_20240830054046_init_postgres.py

```diff
@@ -0,0 +1,75 @@
+from tortoise import BaseDBAsyncClient
```

Comment:
This has to be merged with the initial migration: the very first migration has to create the aerich relation, so that aerich can start tracking the applied migrations. For Postgres, this is not the case if only the second migration creates the table.

Reply:
I see. That is how I had it in the beginning. Will do, thanks.